libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Util/SFSDKResourceUtils.m

```diff
@@ -54,8 +54,15 @@ + (UIImage *)imageNamed:(NSString *)name
 {
     NSBundle *bundle = [NSBundle bundleForClass:[self class]];
     NSAssert(name != nil, @"name must contain a value.");
+    UIImage *image = [UIImage imageNamed:name inBundle:bundle compatibleWithTraitCollection:nil];
+    if (image) {
+        return image;
+    }
     
-    return [UIImage imageNamed:name inBundle:bundle compatibleWithTraitCollection:nil];
+    // get from main bundle
+    bundle = [NSBundle mainBundle];
```

Comment:
Ah, that's the third case I was missing: the umbrella framework.  I would consider this fix temporary, then, until the resources are properly aligned for the umbrella framework?